mirrord/operator/src/client.rs

```diff
@@ -585,14 +592,13 @@ impl OperatorApi<PreparedClientCert> {
             let namespace = self.target_namespace(layer_config);
             let copied = self
                 .copy_target(
-                    target,
-                    scale_down,
-                    namespace,
-                    layer_config
-                        .feature
-                        .split_queues
-                        .is_set()
-                        .then(|| layer_config.feature.split_queues.clone()),
+                    target, scale_down, namespace,
+                    // layer_config
+                    //     .feature
+                    //     .split_queues
+                    //     .is_set()
+                    //     .then(|| layer_config.feature.split_queues.clone()),
+                    None,
```

Comment:
?

Reply:
This is just what I was using to test the feature, since I don't have a way to use SQS (nor any idea how to mock it).